pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -268,7 +268,7 @@ func (m *manager) AddContainer(pod *v1.Pod, container *v1.Container, containerID
 	m.Lock()
 	defer m.Unlock()
 	if cset, exists := m.state.GetCPUSet(string(pod.UID), container.Name); exists {
-		m.lastUpdateState.SetCPUSet(string(pod.UID), container.Name, cset)
+		m.lastUpdateState.SetCPUSet(string(pod.UID), container.Name, cset, m.state.GetDefaultCPUSet())
```

Comment:
Can we pass it into the struct when initializing it, instead of calling each method once?

Reply:
@googs1025 It look good to me. Should I refactor during this change?

Reply:
@chenk008 yes, please update the PR according to @googs1025's suggestion, thanks!

Reply:
> Can we pass it into the struct when initializing it, instead of calling each method once?

@googs1025 Sorry, I misunderstood before. `AddContainer` is only called from internalContainerLifecycleImpl.PreStartContainer. What you mean by initialization is `manager.reconcileState`?

Reply:
"default" stands for the default value, which should be set at initialization. I don't quite understand why the default value needs to be passed in again when calling each method? This doesn't seem to match the meaning of default

Reply:
`defaultCPUSet` here is meant to represent the CPUset all new containers get by default, which can be further changed later in the flow to handle exclusive CPU assignment. 
It seems to me that passing it explicitely is in the spirit of this change: every time we checkpoint the (exclusive) assignments, we should also checkpoint the defaultCPUset, in the same transaction